### DISCUSSION TITLE
comparing date with each other

### DISCUSSION THREAD

**Discussion by mohammadjavad948** [SeaQL/sea-orm#1000] [Q&A] [answered]
i have a column called `date` on database and i want to get any row that `date` is gte than some other date so im doing this : 
```rust
report::Column::Date.gte(value)
```
when i run query it says: 
`operator does not exist: timestamp without time zone >= text`

how can i do it?

**Comment by mohammadjavad948** [comment, marked as answer]
looks like i can use chrono `DateTime` for it :)

you have to parse your date

```rust
            let value = date.parse::<DateTime<Utc>>()?;

            report::Column::Date.gte(value)

```

**Reply by tyt2y3** [reply]
Actually, it will be helpful if you can also post a snippet of your solution. It might help others )
